validator/testcases/javascript/predefinedentities.py

```diff
@@ -579,13 +579,30 @@
                       u"nsIDOMGeoGeolocation": {"dangerous":
                          "Use of the geolocation API by add-ons requires "
                          "prompting users for consent."},
-                      u"nsIX509CertDB": {"dangerous": {
-                          "description": "Access to the X509 certificate "
-                                         "database is potentially dangerous "
-                                         "and requires careful review by an "
-                                         "administrative reviewer.",
-                          "editors_only": True,
-                      }}}
+                      }
+
+CONTRACT_ENTITIES = {}
```

Comment:
This feels a bit weird, why not remove this line, and on line 600 do something like
`CONTRACT_ENTITIES = dict(....)`